src/connectors/impls/logging.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright 2021, The Tremor Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::connectors::prelude::*;
+use async_broadcast::Receiver;
+use tremor_pipeline::{LoggingMsg, LOGGING_CHANNEL};
+
+use crate::connectors::{prelude::KillSwitch, ConnectorBuilder, ConnectorType};
+
+#[derive()]
+pub(crate) struct PluggableLoggingConnector {
+    rx: Receiver<LoggingMsg>,
+}
+
+impl PluggableLoggingConnector {
+    pub(crate) fn new() -> Self {
+        Self {
+            rx: LOGGING_CHANNEL.rx(),
+        }
+    }
+}
+
+#[derive(Debug, Default)]
+pub(crate) struct Builder {}
+
+#[async_trait::async_trait]
+impl ConnectorBuilder for Builder {
+    fn connector_type(&self) -> ConnectorType {
+        "logging".into()
```

Comment:
We call the `metrics` connector `metrics` to follow the same naming scheme for consistency would mean `logs` is a better name for this connector.

Reply:
this is named consistently now :) resolving the comment :D